composer.json

```diff
@@ -20,6 +20,12 @@
             "Appwrite\\Tests\\": "tests/extensions"
         }
     },
+    "repositories": [
```

Comment:
Reminder to remove this part after we register the lib on Packagist.

Reply:
Removed.

Reply:
Put it back in order to fix Travis tests. Reopened this issue, will remove after publishing logger library.

Reply:
Package is now available:
https://packagist.org/packages/utopia-php/logger

Reply:
Removed repositories array